pkg/label/label.go

```diff
@@ -24,6 +24,11 @@ import (
 const (
 	// The following labels are recommended by kubernetes https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
 
+	// ResourceNameLabelKey is the unique identifier of the resource, its value is the same as metadata.name
+	ResourceNameLabelKey string = "kubernetes.io/name"
```

Comment:
if we have a label to identify resources by purposes, is this label necessary? users can always get resources by name.

Reply:
will only used by user want use label identify a service, but this doesn’t necessary. deleted